pyproject.toml

```diff
@@ -27,6 +24,7 @@ Jinja2 = "^2.11"
 Markdown = "^3.3"
 MarkupSafe = "^1.1"
 mkdocs = "^1.1"
+mkdocs-autorefs = { git = "https://github.com/mkdocstrings/autorefs.git" }
```

Comment:
This should be pointing at the pypi release. Please make one when you're ready.

Reply:
Done!